dataactvalidator/interfaces/interfaceHolder.py

```diff
@@ -8,10 +9,17 @@ class InterfaceHolder:
 
     def __init__(self):
         """ Create the interfaces """
-        self.jobDb = ValidatorJobTrackerInterface()
-        self.errorDb = ValidatorErrorInterface()
-        self.stagingDb = ValidatorStagingInterface()
-        self.validationDb = ValidatorValidationInterface()
+        if BaseInterface.interfaces is None:
+            self.jobDb = ValidatorJobTrackerInterface()
+            self.errorDb = ValidatorErrorInterface()
+            self.stagingDb = ValidatorStagingInterface()
+            self.validationDb = ValidatorValidationInterface()
+            BaseInterface.interfaces = self
```

Comment:
The validator and broker were initially much more separate than they are now.  While it's still the case that they won't both be running in the same process, I've been thinking for awhile that we should combine these two in some way, the only issue is that the individual interfaces are still different for validator and broker.  I'm thinking I could create a common base class in the core package that has most of the functionality, and rename the existing ones to ValidatorInterfaceHolder and BrokerInterfaceHolder to match our conventions for individual interfaces.  The subclasses would then only be responsible for passing the list of interface objects to be used up to the base class.  Thoughts?